lib/msal-common/src/app/config/MsalConfiguration.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ICacheStorage } from "../../cache/ICacheStorage";
+import { INetworkModule } from "../../network/INetworkModule";
+import { ICrypto, PKCECodes } from "../../utils/crypto/ICrypto";
+
+/**
+ * Use the configuration object to configure MSAL Modules and initialize the base interfaces for MSAL.
+ *
+ * This object allows you to configure important elements of MSAL functionality:
+ * - storage: this is where you configure storage implementation.
+ * - network: this is where you can configure network implementation.
+ * - crypto: implementation of crypto functions
+ */
+export type MsalConfiguration = {
+    storageInterface: ICacheStorage,
+    networkInterface: INetworkModule,
+    cryptoInterface: ICrypto
+};
+
+const DEFAULT_STORAGE_OPTIONS: ICacheStorage = {
+    clear: () => {
+        console.log("clear() has not been implemented for the cacheStorage interface.");
```

Comment:
might want to do console.warn instead of log

Reply:
done